webapp/src/widgets/menu/textOption.tsx

```diff
@@ -23,6 +23,10 @@ function TextOption(props:TextOptionProps): JSX.Element {
                 e.target.dispatchEvent(new Event('menuItemClicked'))
                 props.onClick(props.id)
             }}
+            onMouseDown={(e: React.MouseEvent) => {
+                e.target.dispatchEvent(new Event('menuItemClicked'))
+                props.onClick(props.id)
+            }}
```

Comment:
Note: 

I had to add this because the `onBlur` handler on the property name was being called over the onClick when I clicked on a new property Type.

Reply:
Maybe other option is to make a debounced version of the `onTypeAndNameChanged` so the last one is the one that is used. That way you don't have to mess with this `onMouseDown`.

Reply:
Done!